build.sbt

```
@@ -960,7 +960,7 @@ lazy val sbtTest = project.in(file("test") / "sbt-test")
     sbtTestDirectory := baseDirectory.value,
 
     scriptedBatchExecution := true, // set to `false` to execute each test in a separate sbt instance
-    scriptedParallelInstances := 2, // default is 1
+    scriptedParallelInstances := 1, // default is 1, was 2
```

```suggestion
    scriptedParallelInstances := (if (insideCI.value) 1 else 2), // default is 1; races cause spurious failures
```

Running `sbtTest/scripted` locally went from 137s to 216s with this change, definitely OK for CI if it avoids the races.